src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -363,6 +363,7 @@
             "resultETA": "Result ETA: {{eta}}",
             "resultETATooltip": "Shows how long an average submission must wait for its result. Depends on the current utilization of the build system.",
             "titleTooltip": "The title of the exercise can always be changed after creating it. It will be part of the project name of the for the exercise created project on the VCS server",
+            "channelNameTooltip": "Channel Name tooltip",
```

Comment:
```suggestion
            "channelNameTooltip": "A course-wide channel will be created automatically with this name. It can be changed any time.",
```